src/api/crypto/Seed.cpp

```diff
@@ -402,9 +402,7 @@ auto Seed::GetPaymentCode(
     const auto& api = asymmetric_.Internal().API();
     auto code = factory_.Secret(0);
     api.Crypto().Hash().Digest(
-        opentxs::crypto::HashType::Sha256D,
-        key.PublicKey(),
-        code->WriteInto());
+        opentxs::crypto::HashType::Sha256D, key.PublicKey(), code->WriteInto());
```

Comment:
is it automatic change done by otscript formater or Yours IDE ?

Reply:
probably by ./format script